cluster/addons/fluentd-elasticsearch/fluentd-es-image/td-agent.conf

```diff
@@ -105,11 +105,13 @@
   type null
 </match>
 
+# Example:
+# {"log":"[info:2016-02-16T16:04:05.930-08:00] Some log text here\n","stream":"stdout","time":"2016-02-17T00:04:05.931087621Z"}
 <source>
   type tail
   path /var/log/containers/*.log
   pos_file /var/log/es-containers.log.pos
-  time_format %Y-%m-%dT%H:%M:%S
+  time_format %Y-%m-%dT%H:%M:%S.%N
```

Comment:
I think the correct form would be %NZ in this case, given the time field seems to be outputting timezone abbreviation rather than offset from UTC.  Does that sound correct?